panel.py

```diff
@@ -16,7 +16,7 @@ def __init__(self,w,height,width):
                 self.w = w
                 self.height = height
                 self.width = width
-                self.activities = ['Activity 1', 'Activity 2', 'Activity 3', 'Activity 4', 'Activity 5','Activity 6']
+                self.activities = [['Activity 1',IntVar()], ['Activity 2',IntVar()], ['Activity 3',IntVar()], ['Activity 4',IntVar()], ['Activity 5',IntVar()],['Activity 6',IntVar()]]
```

Comment:
What does IntVar() do ?

Reply:
IntVar() is a tk variable tat is either 1 or 0

Reply:
Oh ok cool ! Does IntVar() can hold multiple 1s and 0s ? since each activity has 7 of them...

Reply:
just one i think, each activity has its own